scripts/migrate_score_override.py

```diff
@@ -0,0 +1,138 @@
+import itertools
+from django.contrib.auth.models import User
+from django.utils.timezone import now
+from opaque_keys.edx.locations import SlashSeparatedCourseKey
+from openassessment.assessment.models import Assessment, AssessmentPart, StaffWorkflow
+from openassessment.workflow.models import AssessmentWorkflow, AssessmentWorkflowStep
+from student.models import anonymous_id_for_user
+from submissions.models import Score, ScoreAnnotation, Submission
+
+old_scores = Score.objects.filter(submission__isnull=True, reset=False)
+for score in old_scores:
+    points_override = score.points_earned
+    points_possible = score.points_possible
+
+    # Update old override with new reset
+    score.points_earned = 0
+    score.points_possible = 0
+    score.reset = True
+    score.save()
```

Comment:
Doing this block here means if something goes wrong further down then the program cannot be run again and data has been lost.